src/DocRaptor.php

```diff
@@ -122,11 +140,21 @@ public function setTag($tag)
      */
     public function setTest($test = false)
     {
-        $this->test = (bool)$test;
+        // Strict type check, needs to be conservative
```

Comment:
Yeah, probably not, they're more like explanations why casting to bool isn't the best idea. Will remove them.